framework/src/application/node/node.js

```diff
@@ -596,14 +584,6 @@ module.exports = class Node {
 			},
 		);
 
-		this.transactionPool.on(EVENT_UNCONFIRMED_TRANSACTION, transaction => {
-			this.logger.trace(
-				{ transactionId: transaction.id },
-				'Received EVENT_UNCONFIRMED_TRANSACTION',
-			);
-			this.transport.handleBroadcastTransaction(transaction);
```

Comment:
where do we call `handleBroadcastTransaction` now?

Reply:
🤔 .. we missed the emit event and handling requirement... when we add a transaction to pool.. we should broadcast..

Reply:
it was already implemented 😉